revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
+                    color: white;
+                    text-align: center;
+                    transition: all 0.3s;
+                }
+            }
+
+            &:hover {
+                background-color: white;
+
+                p {
+                    color: black;
+                    font-weight: 500;
+                }
+            }
+        }
+
+        .project-title {
+            color: white;
+            font-size: 2rem;
+            font-weight: 800;
+            top: 400px;
+            left: 5%;
+            position: absolute;
+            width: 30%;
+            line-height: 2rem;
+
+        }
+    }
+}
+
+.dashboard {
+    height: 400px;
+    z-index: 1000;
+    position: absolute;
+    top: 300px;
+
+    .graphics-container {
+        height: 100%;
+        width: 100%;
+
+        .top-bar {
+            background-color: $revolv-blue;
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 50%;
+            display: inline-block;
+            margin: 0;
+
+            .container {
+                position: relative;
+                color: white;
+                font-weight: 500;
+                width: 100%;
+                height: 100%;
+                margin: 0;
+                display: inline-block;
+
+                .amount-donated {
+                    float: left;
+                    width: 30%;
+                    height: 100%;
+                    line-height: 40px;
+                    margin: 0px 0px 0px 20px;
+                    font-size: 1.4rem;
+                }
+
+                .days-left {
+                    float: right;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 20px 0px 0px;
+                }
+            }
+        }
+
+        .bottom-bar {
+            background-color: #064f7e;
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 60%;
+            display: inline-block;
+
+            .container {
+                position: relative;
+                color: white;
+                font-weight: 500;
+                width: 100%;
+                height: 100%;
+                margin: 0;
+
+                .amount-saved {
+                    float: left;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 0px 0px 20px;
+                }
+
+                .num-donors {
+                    float: right;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 20px 0px 0px;
+                }
+            }
+        }
+
+        .svg-graphics-container {
+            height: 100%;
+            width: 100%;
+            position: absolute;
+        }
+
+        .internal-graphics-container {
```

Comment:
Why do you need both `svg-graphics-container` and `internal-graphics-container`?